not-so-smart-contracts/algorand/rekeying/README.md

```diff
@@ -13,6 +13,7 @@ Contract accounts are accounts which are derived from the Teal code that is in c
 
 Similar issue affects the accounts that created a delegate signature by signing a Teal program. Delegator is only needed to sign the contract and any user with access to delegate signature can construct and submit transactions. Because of this, a malicious user can rekey the sender’s account if the Teal logic accepts a transaction with the rekey-to field set to the user controlled address.
 
+Note: From Teal v6, Applications can also be rekeyed by executing an inner transaction with "RekeyTo" field set to a non-zero address. Rekeying an application allows to bypass the application logic and directly transfer Algos and assets of the application account.
```

Comment:
Minor: `Applications` should be replaced by `Application accounts`, as applications are not accounts on Algorand.